### PR TITLE
test(lld): fix & generate new playwright screenshots

### PR DESCRIPTION
**Pull request by ofreyssinet-ledger**
### ✅ Checklist

- [ ] `npx changeset` was attached. -> **No functional change**
- [x] **Covered by automatic tests.** 
- [ ] **Impact of the changes:** 
  - no functional change

### 📝 Description

The playwright screenshot for the test of the "install set of apps" (component of the sync onboarding) was too large, now it takes the component under test itself instead of the whole page.

### ❓ Context

- **JIRA or GitHub link**: 

---

### 🧐 Checklist for the PR Reviewers

- **The code aligns with the requirements** described in the linked JIRA or GitHub issue.
- **The PR description clearly documents the changes** made and explains any technical trade-offs or design decisions.
- **There are no undocumented trade-offs**, technical debt, or maintainability issues.
- **The PR has been tested** thoroughly, and any potential edge cases have been considered and handled.
- **Any new dependencies** have been justified and documented.
- **Performance** considerations have been taken into account. (changes have been profiled or benchmarked if necessary)

### CODE REVIEW

**Review by ypolishchuk-ledger — APPROVED** [review]

File: apps/ledger-live-desktop/tests/specs/onboarding/connect-device.spec.ts [hunk 1]
```diff
@@ -25,6 +25,7 @@ test.describe.parallel("Onboarding", () => {
       await test.step("Get started", async () => {
         await onboardingPage.getStarted();
         await onboardingPage.hoverDevice(Nano.nanoS);
+        await new Promise(r => setTimeout(r, 300)); // wait for the animation to finish
```
Comment (KVNLS) [inline apps/ledger-live-desktop/tests/specs/onboarding/connect-device.spec.ts:28]: We should be able to deactivate animations for e2e tests

Comment (ofreyssinet-ledger) [inline apps/ledger-live-desktop/tests/specs/onboarding/connect-device.spec.ts:28]: before this change, I ran /generate-screenshots and it generated a screenshots where the hover wasn't visible

**Review by KVNLS — APPROVED** [review]

File: apps/ledger-live-desktop/tests/specs/onboarding/connect-device.spec.ts (hunk 1 above)
Comment (valpinkman) [inline apps/ledger-live-desktop/tests/specs/onboarding/connect-device.spec.ts:28]: not sure about this. maybe there is a `delay` helper if i'm not mistaking. Also, when running with playwright i think we disable the animations ? Maybe someone from the hub team can give us more infos,

**Pull request merged**